.github/workflows/release.yml

```diff
@@ -0,0 +1,113 @@
+name: Build/release Zeek
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+    tags:
+      - v*brim*
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    strategy:
+      matrix:
+        platform: [macos-12, ubuntu-20.04, windows-2019]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - name: Checkout build-zeek
+      uses: actions/checkout@v3
+
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.21
+
+    - name: Checkout zeek
+      uses: actions/checkout@v3
+      with:
+        repository: zeek/zeek
+        ref: v6.0.2
+        fetch-depth: 1
+        submodules: recursive
+        path: zeek-src
+
+    - name: Build zeekrunner (Windows)
+      if: startsWith(matrix.platform, 'windows-')
+      run: go build -o zeekrunner.exe zeekrunner.go
+
+    # Includes workaround from https://github.com/maxmind/libmaxminddb/pull/334
+    - name: Build libmaxminddb (Windows)
+      if: startsWith(matrix.platform, 'windows-')
+      run: |
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
+        curl -L https://github.com/maxmind/libmaxminddb/releases/download/1.8.0/libmaxminddb-1.8.0.tar.gz | tar xzvf -
+        mkdir libmaxminddb-1.8.0\build
+        cd libmaxminddb-1.8.0\build
+        cmake ..
+        cmake --build .
+        cmake --build . --target install
+        rename "C:\Program Files (x86)\maxminddb\include\maxminddb.h" maxminddb.h.bak
+        sed "/typedef ADDRESS_FAMILY sa_family_t/d" "C:\Program Files (x86)\maxminddb\include\maxminddb.h.bak" > "C:\Program Files (x86)\maxminddb\include\maxminddb.h"
```

Comment:
I did this in two steps rather than `sed -i` because on the Windows runner that failed with a `cannot rename: Invalid cross-device link` error referencing a temp directory on a different drive letter. This workaround is all temporary until they release another libmaxminddb, so I chose not to obsess over that.